### PR TITLE
implement UsersCounter

### PR DESCRIPTION
**Pull request by R1ZZU**
screen shot 2017-11-26 at 1 46 37 am

### CODE REVIEW

File: src/components/UsersCounter/UsersCounter.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            .count {
+                font-size: 1.5em;
+                margin-right: 5px;
+            }
+
+            .label {
+                display: flex;
+                flex-direction: column;
+                font-size: 0.6em;
+                align-items: center;
```
Comment (kirilknysh) [inline src/components/UsersCounter/UsersCounter.js]: Shouldn't it be left-aligned?
initial css design:
image

_qd-ui:
image

**Comment by DaQuirm** [comment]
Are we still using the same font for joined/solved?

**Comment by R1ZZU** [comment]
font-family or color?

**Comment by DaQuirm** [comment]
font-family. well, both really

On Sun, Nov 26, 2017 at 12:25 PM, Andrei Volchenko <notifications@github.com

**Comment by R1ZZU** [comment]
checked in legacy cssqd game-master. font-family is the same, color is to be changed in parent component

**Comment by DaQuirm** [comment]
Awesome, thanks

**Pull request merged**